ramls/dataImport.raml

```diff
@@ -309,3 +309,23 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
+  /splitStatus:
```

Comment:
please, add the newly defined endpoint to the ModuleDescriptor-template.json file so that it will be available

Reply:
Eric, propose to make the endpoint path in hyphen-separated (split-status) format instead of camel case format just ensure compliance with project conventions https://dev.folio.org/guidelines/naming-conventions/#:~:text=REST%20JSON%20interfaces.-,API%20endpoints,-The%20back%2Dend
Thanks!

Reply:
I added an entry to the descriptor and gave it a permission for access.